src/Main.java

```diff
@@ -21,22 +65,30 @@ public static void main(String[] args) {
      */
     public static void feedback(String stringnumber) {
         Scanner scanner = new Scanner(System.in);
-        StringBuilder feedback = new StringBuilder();
-        System.out.println("take a guess");
-        String guess = scanner.nextLine();
-        if (Objects.equals(guess, stringnumber)) {
-            System.out.println("gefeliciteerd je hebt het goed");
-        } else {
-            for (int i = 0; i < 4; i++) {
-                if (guess.substring(i, i + 1).equals(stringnumber.substring(i, i + 1))) {
-                    feedback.append("+");
-                } else if (stringnumber.contains(guess.substring(i, i + 1))) {
-                    feedback.append("0");
-                } else {
-                    feedback.append("X");
+
+        /* bonus-bonus: in een while loop geplaatst om meerdere keren te kunnen raden */
+        boolean play = true;
+        while (play) {
+            StringBuilder feedback = new StringBuilder();
+            System.out.println("take a guess");
```

Comment:
Je hebt het volgens helemaal gedaan zoals de readme, maar als user begrijp ik niet wat ik moet doen als er take a guess staat. Dus misschien handiger even een intro oid te maken, dat mensen weten wat voor guess ze moeten doen :) 

Als ik een verkeerde guess do (bijvoorbeeld 1 letter invul - of teveel/weinig cijfers) - dan krijg ik error codes van Java. Dit kan je met een try and catch ondervangen. 
Ik heb dat bijvoorbeeld zo opgelost (geen idee of dit de juiste manier is, maar volgens mij werkt het): 

`public static void feedback(String stringNumber) {
        Scanner scanner = new Scanner(System.in);
        StringBuilder feedback = new StringBuilder();
        System.out.println("Kies een getal van 4 unieke cijfers om de mastermind code te raden.");
        boolean continueTrying = true;
        while (continueTrying) {
            String guess = scanner.nextLine();
            if (guess.length() != 4) {
                System.out.println("Je antwoord moet bestaan uit 4 cijfers, niet meer, niet minder. \n Probeer nogmaals: ");
                continue;
            } else if (Objects.equals(guess, stringNumber)) {
                System.out.println("Gefeliciteerd je hebt het goed geraden");
                continueTrying = false;
                break;
            } else {
                for (int i = 0; i < 4; i++) {
                    if (guess.substring(i, i + 1).equals(stringNumber.substring(i, i + 1))) {
                        feedback.append("+");
                    } else if (stringNumber.contains(guess.substring(i, i + 1))) {
                        feedback.append("0");
                    } else {
                        feedback.append("X");
                    }
                }
                System.out.println(feedback.toString());
                System.out.println("Helaas, verkeerd geraden. \n + betekent dat het cijfer juist is en op de juiste plek staat. \n O betekent een juist cijfer, maar verkeerde plek. \n X betekent dat het cijfer er niet in zit. \nProbeer nogmaals:");
                feedback = new StringBuilder();
                continue;

            }
        }
    }`